cmake/AddChmTarget.cmake

```diff
@@ -52,7 +52,6 @@ function (add_chm_target docname lang entities figures dtd_files)
         OUTPUT "${BUILD_DIR}/htmlhelp.hhp"
         COMMAND ${XSLTPROC} ${XSLTPROCFLAGS} ${XSLTPROCFLAGS_CHM}
                 -o "${BUILD_DIR}/"
-                --path "${CMAKE_SOURCE_DIR}/docbook"
```

Comment:
If this is no longer needed here, it should probably have been removed the moment the docbook changes were made.